src/ImportTimerTrigger/FitOnFhir.ImportTimerTrigger/Startup.cs

```diff
@@ -3,28 +3,28 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
-using System;
+using FitOnFhir.Common;
+using FitOnFhir.Common.Config;
 using FitOnFhir.Common.Persistence;
 using FitOnFhir.Common.Repositories;
 using FitOnFhir.GoogleFit.Repositories;
 using FitOnFhir.ImportTimerTrigger;
 using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Health.Common.DependencyInjection;
 
 [assembly: FunctionsStartup(typeof(Startup))]
 
 namespace FitOnFhir.ImportTimerTrigger
 {
-    public class Startup : FunctionsStartup
+    public class Startup : StartupBase
     {
-        public override void Configure(IFunctionsHostBuilder builder)
+        public override void Configure(IFunctionsHostBuilder builder, IConfiguration configuration)
         {
-            string storageAccountConnectionString = Environment.GetEnvironmentVariable("AzureWebJobsStorage");
-
-            builder.Services.AddSingleton(sp => new StorageAccountContext(storageAccountConnectionString));
+            builder.Services.AddConfiguration<AzureConfiguration>(configuration);
```

Comment:
It makes sense to me to put these 3 items in the StartupBase class because they are used in all 3 functions - At the very least, I think AzureConfiguration belongs in the base class.

```c#
builder.Services.AddConfiguration<AzureConfiguration>(configuration);
builder.Services.AddSingleton<IUsersTableRepository, UsersTableRepository>();
builder.Services.AddSingleton<IGoogleFitUserTableRepository, GoogleFitUserTableRepository>();
```

Reply:
It makes sense to move `AzureConfiguration ` and `UsersTableRepository` to `StartupBase`, but I'm inclined to keep `GoogleFitUserTableRepository` out because it is platform specific.  I can try to move `GoogleFitUserTableRepository` to a `GoogleFitStartupBase` class that inherits from the platform agnostic `StartupBase` instead.